src/index.ts

```diff
@@ -22,29 +22,74 @@ interface TelescopeNode {
   telescopicOut: TelescopicOutput;
 }
 
+/**
+ * Modes that designate what form the input text is in and should be interpreted as.
+ */
+enum TextMode {
+  Text = "text",
+  Html = "html",
+}
+const TextModeValues = Object.values(TextMode);
+
 interface Config {
   /**
-  * Character used to separate entries on the same level. Defaults to a single space (" ")
-  */
+   * Character used to separate entries on the same level. Defaults to a single space (" ")
+   */
   separator?: string;
   /**
-  * If true, allows sections to expand automatically on mouse over rather than requiring a click.
-  */
+   * If true, allows sections to expand automatically on mouse over rather than requiring a click. Defaults to false.
+   */
   shouldExpandOnMouseOver?: boolean;
+  /**
+   * A mode that designates what form the input text is in and should be interpreted as. Defaults to 'text'.
+   */
+  textMode?: TextMode;
 }
 
-const DefaultConfig: Config = {
```

Comment:
i moved these into individual defaults because otherwise you wouldn't get the proper default if you passed in a partial config